MAPL_cfio/ShaveMantissa_py.F90

```diff
@@ -0,0 +1,25 @@
+subroutine Shave32 ( a_shaved, a, n, xbits, has_undef, undef, chunksize, rc )
+
+!
+!  Simple cover for f2py.
+!
+   implicit NONE
+   integer,      intent(in) :: n              ! array size
+   real(kind=4), intent(in) :: a(n)           ! array to be shaved, usually 2D
+   integer,      intent(in) :: xbits          ! number of mantissa bits to zero (out of 24)
+   integer,      intent(in) :: has_undef      ! set to 1 if undef is present, 0 otherwise
+   real(kind=4), intent(in) :: undef          ! undef value
+   integer,      intent(in) :: chunksize      ! find mid-range over chunksizes
+
+   real(kind=4), intent(out) :: a_shaved(n)  ! shaved array
+   integer,      intent(out) :: rc            ! error code
```

Comment:
```suggestion
   implicit NONE
   integer,      intent(in) :: n              ! array size
   real(kind=REAL32), intent(in) :: a(n)           ! array to be shaved, usually 2D
   integer,      intent(in) :: xbits          ! number of mantissa bits to zero (out of 24)
   integer,      intent(in) :: has_undef      ! set to 1 if undef is present, 0 otherwise
   real(kind=REAL32), intent(in) :: undef          ! undef value
   integer,      intent(in) :: chunksize      ! find mid-range over chunksizes

   real(kind=REAL32), intent(out) :: a_shaved(n)  ! shaved array
   integer,      intent(out) :: rc            ! error code
```
Not portable.  Probably needs a USE statement for accessing REAL32

Reply:
Okay. It looks like if I also add a `use iso_fortran_env, only: REAL32` it builds. I'll work with @patricia-nasa to make sure if we make that change, it also runs (since I don't know how to 😄 )

Reply:
@patricia-nasa has tested the changed code and it works.